src/upload/system/library/transbank/utils/LogHandler.php

```diff
@@ -0,0 +1,158 @@
+<?php
+namespace Transbank\Utils;
+use Monolog\Logger;
+use Monolog\Handler\RotatingFileHandler;
+
+class LogHandler
+{
+    //constants for log handler
+    const LOG_DEBUG_ENABLED = false; //enable or disable debug logs
+    const LOG_INFO_ENABLED = true; //enable or disable info logs
+    const LOG_ERROR_ENABLED = true; //enable or disable error logs
```

Comment:
delete comments

Reply:
If constants name say enabled, why the value is false?